gluetool/tests/test_error.py

```diff
@@ -3,7 +3,7 @@
 
 import pytest
 from mock import MagicMock
-from hypothesis import example, given, strategies as st
+from hypothesis import assume, example, given, strategies as st
```

Comment:
@happz importing assume but not using it?

Reply:
Right, it is not used anywhere (I had an idea about how to use it, but I chose different way).